src/repository/crud/user.py

```diff
@@ -0,0 +1,20 @@
+from ..models.user import User
+from ..mysql_connection import mysql_connection
+from aiomysql import Error
+
+async def save(user: User):
+    try:
+        query = "INSERT INTO user (username, password, email, full_name, address, phone) VALUES (%s, %s, %s, %s, %s, %s)"
+        user_data = (user.username, user.password, user.email, user.full_name, user.address, user.phone)
+        
+        cursor = mysql_connection.cursor()
+        cursor.execute(query, user_data)
+        mysql_connection.commit()
```

Comment:
here we should be able to save this line with the autocommit to True in the opening of the connection to bd

Reply:
Done.